helm_deploy/laa-court-data-adaptor/templates/service.yaml

```diff
@@ -11,5 +11,9 @@ spec:
       targetPort: http
       protocol: TCP
       name: http
+    - port: {{ .Values.metrics_service.port }}
+      targetPort: metrics
```

Comment:
I thought this wasn't necessary? Apply have 'http' and vcd has '9394'... So it seems no-one is sure what this value is.

Reply:
`{{ .Values.metrics_service.port }}` evaluates to 9394